controllers/topolvm_controller.go

```diff
@@ -322,6 +323,28 @@ func getCsiResizerContainer() *corev1.Container {
 	return csiResizer
 }
 
+func getCsiSnapshoterContainer() *corev1.Container {
+
+	args := []string{
+		fmt.Sprintf("--csi-address=%s", DefaultCSISocket),
+		"--connection-timeout=15s",
+		"--leader-election=false",
```

Comment:
this is false by default we dont need it again here.

Reply:
removed.